bitnami/haproxy/README.md

```diff
@@ -0,0 +1,408 @@
+| `initContainers`                     | Add additional init containers to the haproxy pod(s)                                      | `{}`                 |
+
+### Other Parameters
+
+| Name                    | Description                                          | Value   |
+|-------------------------|------------------------------------------------------|---------|
+| `rbac.create`           | Specifies whether RBAC resources should be created   | `false` |
+| `serviceAccount.create` | Specifies whether a ServiceAccount should be created | `true`  |
+| `serviceAccount.name`   | The name of the ServiceAccount to use.               | `""`    |
+
+The above parameters map to the env variables defined in [bitnami/haproxy](http://github.com/bitnami/bitnami-docker-haproxy). For more information please refer to the [bitnami/haproxy](http://github.com/bitnami/bitnami-docker-haproxy) image documentation.
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```console
+helm install my-release \
+  --set service.type=LoadBalancer \
+    bitnami/haproxy
+```
+
+The above command sets the HAProxy service type as LoadBalancer.
+
+> NOTE: Once this chart is deployed, it is not possible to change the application's access credentials, such as usernames or passwords, using Helm. To change these application credentials after deployment, delete any persistent volumes (PVs) used by the chart and re-deploy it, or use the application's built-in administrative tools if available.
+
+Alternatively, a YAML file that specifies the values for the above parameters can be provided while installing the chart. For example,
+
+```console
+helm install my-release -f values.yaml bitnami/haproxy
+```
+
+> **Tip**: You can use the default [values.yaml](values.yaml)
+
+## Configuration and installation details
+
+### [Rolling VS Immutable tags](https://docs.bitnami.com/containers/how-to/understand-rolling-tags-containers/)
+
+It is strongly recommended to use immutable tags in a production environment. This ensures your deployment does not change automatically if the same tag is updated with a different image.
+
+Bitnami will release a new chart updating its containers if a new version of the main container, significant changes, or critical vulnerabilities exist.
+
+### Configuring HAProxy
+
+By default, HAProxy is deployed with a dummy configuration. You will need to edit the following several values to adapt it to your use case:
+
+First, set the set the configuration to be injected in the `haproxy.cfg` file. Alternatively, you can provide an existing ConfigMap with `haproxy.cfg` by using the `existingConfigmap` value. The example below configures HAProxy to forward all requests to port 8080 to a service called `service1:8080` (which we assume it is accessible from inside the cluster).
```

Comment:
The `existingConfigmap` mention here is confusing, shouldn't it be `configuration`?

Reply:
Improved the wording